### DISCUSSION TITLE
AG Instrument, MULTICHANNEL by default

### DISCUSSION THREAD

**Discussion by Dieguitarro** [apohl79/audiogridder#1142] [Q&A]
Hello everyone, sorry for the English (google translate)

First of all I want to thank Andreas Pohl for such a great plugin, I've been using it for a couple of months now and it's great...

I only have one question right now.

Is there a way to make AG Instrument load multical by default? .... every time I open a project I have to activate all of them and it's exhausting, also for each time the channels are activated the plugin is loaded again and I would like to avoid that.

I appreciate any help or guidance you can give me.

**Comment by apohl79** [comment]
Currently no, I will add a feature request to enable a customizable default behaviour.

> every time I open a project I have to activate all of them and it's exhausting, also for each time the channels are activated the plugin is loaded again and I would like to avoid that.

does the "Enable all" button not work for you?

**Reply by Dieguitarro** [reply]
hello andreas

Thank you very much for the quick answer.

Yes, the "Activate All" button works, but it doesn't save the settings, so every time I open a project, I have to activate it on all existing instances.

Is there a way to fix that it saves the configuration? Or I guess we would have to wait for the next beta :)

**Reply by apohl79** [reply]
>Yes, the "Activate All" button works, but it doesn't save the settings, so every time I open a project, I have to activate it on all existing instances.

understood, I consider this a bug, will log another issue for this. there is no workaround for this right now I guess.

**Reply by Dieguitarro** [reply]
ok andres, thank you very much for answering

I will be waiting for the next update

**Reply by emack999** [reply]
> I consider this a bug, will log another issue for this

I do not have this issue on Windows 11, using both Cubase 12 and Gig Performer. 
Selecting the 'Activate All' button, saving and reloading, all the channels are still activated.

@Dieguitarro Which DAW are you using? Are you Mac or Windows?

**Reply by Dieguitarro** [reply]
> > I consider this a bug, will log another issue for this
> 
> I do not have this issue on Windows 11, using both Cubase 12 and Gig Performer. Selecting the 'Activate All' button, saving and reloading, all the channels are still activated.
> 
> @Dieguitarro Which DAW are you using? Are you Mac or Windows?

Hello emack999

I work in cubase 12 windows 10.

how do you work with gig? Do you use the AG plugin for virtual instruments?

**Reply by mixmonkey** [reply]
Just a bump for this bug fix. Activating all the outputs works, but the status doesn't get saved. Having to activate it every time for every AG instrument is kind of a drag.
Incidentally, your automation implementation wipes the floor with Vienna Ensemble Pro 7, which has some _awful_ issues.
